lively.morphic/world-loading.js

```diff
@@ -312,12 +313,31 @@ export async function interactivelySaveWorld (world, options) {
         const { author: { name: authorName }, timestamp } = newerCommit;
         const overwriteQ = `The current version of world ${name} is not the most recent!\n` +
                        `A newer version by ${authorName} was saved on ` +
-                       `${date.format(new Date(timestamp), 'yyyy-mm-dd HH:MM')}. Overwrite?`;
-        overwrite = await world.confirm(['Version Conflict\n', null, overwriteQ, { fontSize: 16, fontWeight: 'normal' }], { width: 600 });
+                       `${date.format(new Date(timestamp), 'yyyy-mm-dd HH:MM')}. Do you want to merge the worlds?`;
+        overwrite = await world.multipleChoicePrompt(['Version Conflict\n', null, overwriteQ, { fontSize: 16, fontWeight: 'normal' }], { width: 600, choices: ['Take mine', 'Merge mine', 'Manual merge', 'Merge theirs', 'Take theirs'] });
```

Comment:
I wouldn't call it overwrite anymore. (Has to be updated when the variable is declared as well)

```suggestion
        strategy = await world.multipleChoicePrompt(['Version Conflict\n', null, strategyQ, { fontSize: 16, fontWeight: 'normal' }], { width: 600, choices: ['Take mine', 'Merge mine', 'Manual merge', 'Merge theirs', 'Take theirs'] });
```